lib/srv/authhandlers_test.go

```diff
@@ -62,7 +62,7 @@ type mockLoginChecker struct {
 	rbacChecked bool
 }
 
-func (m *mockLoginChecker) canLoginWithRBAC(_ *ssh.Certificate, _ types.CertAuthority, _ string, _ types.Server, _, _ string) error {
+func (m *mockLoginChecker) canLoginWithRBAC(_ *sshca.Identity, _ types.CertAuthority, _ string, _ types.Server, _ string) error {
```

Comment:
Suggestion:

```suggestion
func (m *mockLoginChecker) canLoginWithRBAC(*sshca.Identity, types.CertAuthority, string, types.Server, string) error {
```

Reply:
TIL that it's possible to omit the arg name in methods.  I experimented with this, but I actually find I dislike this syntax.  Leaning toward leaving as-is.  An underscore is a well-understood convention for explicitly ignoring a value.  Omitting a name entirely IMO is less readable and more susceptible to human error.